sklearn/metrics/cluster/tests/test_supervised.py

```diff
@@ -348,3 +348,12 @@ def test_fowlkes_mallows_score_properties():
 def test_mutual_info_score_positive_constant_label(labels_true, labels_pred):
     # non-regression test for #16355
     assert mutual_info_score(labels_true, labels_pred) >= 0
+
+
+def test_check_clustering_error():
+    # Test warning message for continuous values
+    noise = np.random.rand(500)
+    wavelength = np.linspace(0.01, 1, 500) * 1e-6
+
+    with pytest.warns(UserWarning):
+        normalized_mutual_info_score(wavelength, noise)
```

Comment:
The name of this test is `test_check_clustering_error` but we are checking `normalized_mutual_info_score`.

Reply:
`check_clusterings` is called in `normalized_mutual_info_score` and i wrote to the test to insure that the fix the original pull request.